manticore/core/smtlib/constraints.py

```diff
@@ -118,7 +126,8 @@ def __get_related(self, related_to=None):
         # satisfiable one, {}.
         #   In light of the above, the core __get_related logic is currently disabled.
         # if related_to is not None:
-        if False:
+        # feliam: This assumes the previous constraints are already SAT (normal SE forking)
+        if consts.related_constraints and related_to is not None:
```

Comment:
As you noted on slack, this will effectively revert #1674 without fixing #1679. Maybe we can set the default to `False` until that's taken care of?